### PR TITLE
Update convenience PIN

### PR DESCRIPTION
**Pull request by rui0122**
Add some statements.

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ sections:
...
+          While *convenience PIN* provides a convenient way to sign in to Windows, it still use a password for authentication. When correct convenience PIN is provided to Windows, it loads password information from its cache and authenticates the user. Convenience PIN is a mechanism that hides password from users using PIN. Customers using *convenience PINs* should move to **Windows Hello for Business**. New Windows deployments should deploy Windows Hello for Business and not convenience PINs. Microsoft will be deprecating convenience PINs in the future and will publish the date early to ensure customers have adequate lead time to deploy Windows Hello for Business. (Please note that 'PIN' configured fro Windows Hello for Business is not same as 'Convinience PIN'. Convinience PIN refers a PIN configured for Windows Hello.)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:41]:
```suggestion
          While *convenience PIN* provides a convenient way to sign in to Windows, it still uses a password for authentication. When the correct convenience PIN is provided to Windows, it loads password information from its cache and authenticates the user. Convenience PIN is a mechanism that hides passwords from users using a PIN. Customers using *convenience PINs* should move to **Windows Hello for Business**. New Windows deployments should deploy Windows Hello for Business and not convenience PINs. Microsoft will deprecate convenience PINs in the future and will publish the date early to ensure customers have adequate lead time to deploy Windows Hello for Business. (Note that 'PIN' configured for Windows Hello for Business is not the same as 'Convenience PIN'. Convenience PIN refers to a PIN configured for Windows Hello for Business.)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rui0122, kindly commit the suggested changes. Many thanks.

**Comment by paolomatarazzo** [comment]
Hello @rui0122 and thank you for the contribution! The FAQ document is in the process of being completely restructured and should be published soon. For this reason, I take into consideration your feedback but closing this PR.

**Comment by rui0122** [comment]
Thank you for reviewing @JohanFreelancer9 
And thank you for your work @paolomatarazzo I will wait for the publish.

**Pull request closed without merging**